src/irc.py

```diff
@@ -216,7 +225,10 @@ def reload(self) -> None:
             ReloadError: when encountering a SnapError
         """
         try:
-            systemd.service_reload(IRC_BRIDGE_SNAP_NAME)
+            systemd.daemon_reload()
+            systemd.service_enable(IRC_BRIDGE_SERVICE_NAME)
```

Comment:
You are right, I do a daemon_reload and a service_enable first - in case there is no change in the systemd unit files and the service is enabled, that's a no-op. Then I check if the service is running via service_running and invoke a service_start, so if it is running it is again a no-op. I went with that due to the reconcile nature of everything running.

Reply:
mmm but if for example the `bridge_admins` config option is changed, then I do not see how the service will get it. I do not see that it is stopped/reloaded/restarted